ion-schema/src/ion_extension.rs

```diff
@@ -19,25 +18,21 @@ pub(crate) trait ElementExtensions {
 impl ElementExtensions for Element {
     fn as_usize(&self) -> Option<usize> {
         match self.value() {
-            Value::Int(Int::I64(i)) => i.to_usize(),
-            Value::Int(Int::BigInt(i)) => i.to_usize(),
+            Value::Int(i) => i.as_i128()?.to_usize(),
```

Comment:
There's an [`Int::as_usize()`](https://docs.rs/ion-rs/1.0.0-rc.7/ion_rs/struct.Int.html#method.as_usize) method you could use here.